universal-application-tool-0.0.1/test/services/applicant/ApplicantServiceImplTest.java

```diff
@@ -46,10 +39,9 @@
   public void setUp() throws Exception {
     subject = instanceOf(ApplicantServiceImpl.class);
     programService = instanceOf(ProgramServiceImpl.class);
-    questionService = instanceOf(QuestionService.class);
     applicantRepository = instanceOf(ApplicantRepository.class);
-    createQuestions();
-    createProgram();
+    questionDefinition = TestQuestionBank.applicantName().getQuestionDefinition();
```

Comment:
(blocking) the `TestQuestionBank` should be used in the methods that need the question. We don't need to use a class level variable for this.

Reply:
Removed the class level variable

`createProgram(TestQuestionBank.applicantName().getQuestionDefinition());`